maths/average_mode.py

```diff
@@ -0,0 +1,18 @@
+def mode(inputlist):                    					#Defining function "mode."

```

Comment:
Let's change this to `input_list` to be more Pythonic as discussed in CONTRIBUTING.md.